media-api/app/lib/elasticsearch/QueryBuilder.scala

```diff
@@ -26,10 +26,18 @@ class QueryBuilder(matchFields: Seq[String], overQuotaAgencies: () => List[Agenc
   private def multiMatchPhraseQuery(value: String, fields: Seq[String]): MultiMatchQuery =
     ElasticDsl.multiMatchQuery(value).fields(fields).matchType(MultiMatchQueryBuilderType.PHRASE)
 
+  private def multiMatchWordQuery(value: String, fields: Seq[String]): MultiMatchQuery = {
+    val multiMatchQuery = ElasticDsl.multiMatchQuery(value).fields(fields).operator(Operator.AND)
+
+    if (config.fuzzySearchEnabled) {
+      multiMatchQuery.matchType(MultiMatchQueryBuilderType.BEST_FIELDS).fuzziness(config.fuzzySearchEditDistance)
+    } else {
+      multiMatchQuery.matchType(MultiMatchQueryBuilderType.CROSS_FIELDS)
```

Comment:
Why do you switch to best_fields when fuzzy searching is turned on, rather than sticking with cross_fields?

Reply:
@andrew-nowak Elasticsearch (upto version 7.14) does not support fuzzy search on cross_fields multi-match queries.

Reply:
@ochiengolanga
thanks! Just to be sure - have your stakeholders seen and approved the trade-off (that users cannot search across fields once fuzziness is enabled)? We'd be unlikely to turn this on on our side while `cross_fields` can't be used with fuzzy searching, as we don't want to remove functionality that users rely on. 
If you do want this behaviour anyway, can you make sure to clearly document how enabling fuzzy searching changes the search behaviour, including the downsides.